test/Concurrency/Runtime/distributed_actor_remote_functions.swift

```diff
@@ -0,0 +1,149 @@
+// RUN: %target-run-simple-swift(-Xfrontend -enable-experimental-concurrency -Xfrontend -enable-experimental-distributed -parse-as-library) | %FileCheck %s --dump-input=always
+
+// REQUIRES: executable_test
+// REQUIRES: concurrency
+
+import Dispatch
+import _Concurrency
+
+struct Boom: Error {}
+
+distributed actor SomeSpecificDistributedActor {
+  let state: String = "hi there"
+
+  distributed func helloAsyncThrows() async throws -> String {
+    "local(\(#function))"
+  }
+
+//  distributed func helloAsync() async -> String {
```

Comment:
it's a bit sad that uncommenting those now crashes SIL... maybe we can not trigger the remote thing for now for them... 

It'd be nice to show that the typesystem changes implicit throwing/async works hm... Maybe I'll do that for a demo branch